osf/models/mixins.py

```diff
@@ -298,8 +298,13 @@ class AffiliatedInstitutionMixin(models.Model):
     affiliated_institutions = models.ManyToManyField('Institution', related_name='nodes')
 
     def add_affiliated_institution(self, inst, user, save=False, log=True):
+        from .institution import Institution
+        if not isinstance(inst, Institution):
+            raise ValidationValueError(f'Institution not provided')
```

Comment:
I don't think we need to check this. My argument is: if we check this for institution, should we doe the check user too? In addition, we expect people to know what to provide and if they don't, error will be raised when checking/adding affiliation anyway.